canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx

```diff
@@ -191,8 +193,21 @@ class VirtualizedTable extends React.Component {
 		</div>);
 	}
 
-	overSelectOption() {
-		this.isOverSelectOption = !this.isOverSelectOption;
+	overSelectOption(evt) {
+		// Differentiate between mouse and keyboard events
+		if (evt.type === "mouseenter" && this.keyBoardEventCalled === false) {
```

Comment:
Instead of `this.keyBoardEventCalled === false` you can shorten it to `!this.keyBoardEventCalled`, similar for testing true